src/ui/public/timepicker/timepicker.js

```diff
@@ -103,15 +103,15 @@ module.directive('kbnTimepicker', function (refreshIntervals) {
           return;
         }
 
-        _.set($scope, 'browserAbsolute.from', new Date(newDate.year(), newDate.month(), newDate.date()));
```

Comment:
This change and the one below will reintroduce https://github.com/elastic/kibana/issues/5370 which was fixed by https://github.com/elastic/kibana/pull/11773.